cloudbuild.yaml

```diff
@@ -3,17 +3,62 @@
 
 steps:
 # Perform maven build, omitting local docker operations
-- name: 'maven:3.3.9-jdk-8'
-  args: ['mvn', '-B', '-P-local-docker-build', '-P-test.local', '-Ddocker.tag.long=$_DOCKER_TAG', 'clean', 'install']
+- name: 'gcr.io/cloud-builders/java/mvn:3.5.0-jdk-8'
+  args:
+    - '--batch-mode'
+    - '-P-local-docker-build'
+    - '-P-test.local'
+    - '-Ddocker.tag.long=${_DOCKER_TAG}'
```

Comment:
@cassand I thought about this some more, and I actually think this is OK for now. In general, the cloudbuild execution has full control over the way the image is named and tagged, so regardless of what we pass into maven, these fields are still able to be overridden. 

I agree that we should come up with a more principled way of handling these duplicated properties, but I think a better time to handle it is when we start releasing more than one image from this repository (the next major version of Java and Jetty). Is that OK with you?